ui/component/core/src/index.ts

```diff
@@ -40,6 +40,8 @@ export declare type Keycloak = {
     resourceAccess: any;
     onAuthSuccess: () => void;
     onAuthError: () => void;
+    onAuthRefreshSuccess: () => void;
+    onAuthRefreshError: () => void;
```

Comment:
What are these for?

Reply:
It's a callback event of the Keycloak js adapter, which I added to the `Keycloak` type we declare in this file.
(since the adapter doesn't support types out of the box)

FYI: The `_keycloak` object is `protected` here, so these methods are only used in the core module.

Reply:
Ah yes I should have expanded the whole file :)